### DISCUSSION TITLE
Personal question

### DISCUSSION THREAD

**Discussion by nadongjin** [leondgarse/Keras_insightface#72] [Q&A]
Hello
Happy New Year!
This is a personal question.
What architecture should we use to make facial recognition accuracy sota performance?
Based on MS1MV3 data, EfficientNetV2S or EffV2S, and AdamW's architecture are small in size, so the information speed is good, but the accuracy is slightly poor.
Personally, I would like to ask you what architecture is good to use to improve accuracy even if the speed is slightly slow.
Is r100 the best choice? ^^;
Thank you always for your kind reply.

**Comment by leondgarse** [comment]
I‘m mostly trying efficientnetv2 recently. Testing a strategy parameters on `EffV2B0` then apply to `EffV2S`. As far as I tested, `EffV2S` is slightly better than `r50` on `MS1MV3`:
- My `EfficientNetV2S + MS1MV3 + SGD` IJBB 95.6475, IJBC 96.8605.
- My `se_r50 swish SD + MS1MV3 + SGD` IJBB 95.6378, IJBC 96.8144.
- [insightface model_zoo](https://github.com/deepinsight/insightface/tree/master/model_zoo) `IResNet-50 + MS1MV3` IJBC 96.580.

`r100` is another level on model size, so comparing with `r50` here.
But generally talking which is better, IMO `EffcientNet`s are good ones in most CV transfer learning missions, while `ResNet` family models have a good potential enhancing with other structures like `se` / `bottleneck attention` / `halo attention` or others. Like I have a `ResNetV250` like model using `anti alias` down sample instead of `MaxPooling` and `PReLU` activation trained using `MS1MV3 + SGD` reached `IJBC 96.8656`, but it's still not a stable architecture.
Further, there are also other models for CV missions like `ConvNeXt` / `CoAtNet` and many others can be found in [image-classification-on-imagenet](https://paperswithcode.com/sota/image-classification-on-imagenet).
Currently, I would just recommend `EfficientNet`s till found some others better. Yes, `r100` is still better than `Effv2S`, and my testing on `EffV2M` not as good as expect. Using `MS1MV3 + SGD` trained 50 epochs, `Effv2M` is `IJBB 0.95667, IJBC 0.969781`, just a slightly better than `V2S`.
